packages/wallet-sdk/src/core/communicator/Communicator.ts

```diff
@@ -1,10 +1,10 @@
-import { VERSION } from '../../sdk-info.js';
-import { ConfigMessage } from '../message/ConfigMessage.js';
-import { Message, MessageID } from '../message/Message.js';
 import { CB_KEYS_URL } from ':core/constants.js';
 import { standardErrors } from ':core/error/errors.js';
 import { AppMetadata, Preference } from ':core/provider/interface.js';
 import { closePopup, openPopup } from ':util/web.js';
+import { VERSION } from '../../sdk-info.js';
+import { ConfigMessage } from '../message/ConfigMessage.js';
+import { Message, MessageID } from '../message/Message.js';
```

Comment:
wonder why this keeps moving. 😓 Seems like a biome issue

Reply:
reverted for now